src/main/kotlin/no/nav/familie/ef/sak/simulering/SimuleringService.kt

```diff
@@ -83,6 +83,16 @@ class SimuleringService(
         )
     }
 
+    fun simuleringOppsummeringerErUlike(saksbehandling: Saksbehandling): Boolean {
+        val lagretSimuleringoppsummering = hentLagretSimuleringsoppsummering(saksbehandling.id)
+        val simulertTilkjentYtelse = simulerMedTilkjentYtelse(saksbehandling).oppsummering
```

Comment:
```suggestion
        val nySimuleringsoppsummering = simulerMedTilkjentYtelse(saksbehandling).oppsummering
```

For å tydeligjøre at man sammenligner ny vs. gammel simmuleringsoppsummering